packages/clerk-js/src/core/clerk.ts

```diff
@@ -739,6 +741,48 @@ export class Clerk implements ClerkInterface {
     void this.#componentControls.ensureMounted().then(controls => controls.closeModal('userVerification'));
   };
 
+  public __internal_attemptToEnableEnvironmentSetting = (
+    params: __internal_AttemptToEnableEnvironmentSettingParams,
+  ): { status: 'enabled' | 'prompt-shown' } => {
+    const { for: setting, caller } = params;
+
+    // If not in development instance, return enabled status in order to not open the prompt
+    if (this.#instanceType !== 'development') {
+      return { status: 'enabled' };
+    }
+
+    switch (setting) {
+      case 'organizations':
+        if (!disabledOrganizationsFeature(this, this.environment)) {
+          return { status: 'enabled' };
+        }
+
+        this.__internal_openEnableOrganizationsPrompt({
+          caller,
+          // Reload current window to all invalidate all resources
+          // related to organizations, eg: roles
+          onSuccess: () => window.location.reload(),
```

Comment:
There's still some delay in which the window is reloading and the screen kepts showing a blank state without having the component mounted due to the stale environment data.

I'm thinking of applying an optimistic update, at least on the "enabled" status, to have the components mounted meanwhile, or not closing the modal and introduce a loading status until the window gets reloaded.